.github/workflows/jira-sync.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           description: "${{ github.event.issue.body || github.event.pull_request.body }}\n\n_Created in GitHub by ${{ github.actor }}._"
           # customfield_10089 is "Issue Link", customfield_10371 is "Source" (use JIRA API to retrieve), customerfield_10091 is "Team (R&D)
           extraFields: '{ "customfield_10089": "${{ github.event.issue.html_url || github.event.pull_request.html_url }}",
-                          "Team[Team]": "72e166fb-d26c-4a61-b0de-7a290d91708f",
+                          "Team": "72e166fb-d26c-4a61-b0de-7a290d91708f",
```

Comment:
this seems to be the structure of the thing we're targeting, based on an API response I got from a newly created issue:

```
    "customfield_10001": {
      "id": "72e166fb-d26c-4a61-b0de-7a290d91708f",
      "name": "Nomad Minor",
      "avatarUrl": "",
      "isVisible": true,
      "isVerified": false,
      "title": "Nomad Minor",
      "isShared": true
    },
```

I have no idea how much of that we need to specify here, but I bet we can experiment using the Jira API directly, then port that over to this workflow.

the github action that this uses
https://github.com/tomhjp/gh-action-jira-create/blob/main/README.md#inputs

points to the jira api:
https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issues/#api-rest-api-3-issue-post